include/seqan/gff_io/gff_io_base.h

```diff
@@ -349,9 +349,12 @@ void readRecord(GffRecord & record, CharString & buffer, TFwdIterator & iter)
 {
     IsNewline isNewline;
 
-    clear(record);
+    // skip commented lines
+    while(!atEnd(iter) && value(iter) == '#')
```

Comment:
Please mind our C++ style guide (online on the readthedocs site) and consider to use uncrustify with our config (uncrustify.conf)